src/analytics/message_info.rs

```diff
@@ -11,9 +11,10 @@ pub struct MessageInfo {
     pub client_id: Arc<str>,
     pub topic: Arc<str>,
     pub push_provider: Arc<str>,
-    pub always_encrypted: bool,
-    pub encrypted: bool,
-    pub flags: u32,
+    pub always_raw: Option<bool>,
+    pub tag: Option<u32>,
+    pub encrypted: Option<bool>,
```

Comment:
We can't change the scheme since there is already data that's been exported using it. We'd have to wipe the data or adjust queries to exclude the old schema